chia/cmds/wallet.py

```diff
@@ -1406,26 +1326,17 @@ def vcs_cmd():  # pragma: no cover
 @click.option("-f", "--fingerprint", help="Set the fingerprint to specify which key to use", type=int)
 @click.option("-d", "--did", help="The DID of the VC's proof provider", type=str, required=True)
 @click.option("-t", "--target-address", help="The address to send the VC to once it's minted", type=str, required=False)
-@click.option("-m", "--fee", help="Blockchain fee for mint transaction, in XCH", type=str, required=False)
-def _mint_vc(
+@click.option("-m", "--fee", help="Blockchain fee for mint transaction, in XCH", type=str, required=False, default="0")
+def mint_vc_cmd(
     wallet_rpc_port: Optional[int],
     fingerprint: int,
     did: str,
     target_address: Optional[str],
-    fee: Optional[str],
+    fee: str,
```

Comment:
for future PRs, I think it's best to leave unrelated changes, like this one, for separate PRs. including the renaming of functions. it makes reviewing simpler.

Reply:
mypy also made me do this